# 06 Cushing

Does a CUSHING build versus its own prior 52 weeks show up as a lower RWTC print on the nominal EIA Wednesday?

Holdout starts 2020-01-01. Verdict: failed.

This study reads stored series. It does not add a derived column.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'cushing')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


failed
Train correlation of surprise with the release-day RWTC log return is -0.064. Holdout correlation is 0.151. The holdout correlation is not negative. The bar was missed.
CUSHING surprise versus RWTC release-day log return
Train, release before 2020-01-01 | 630 | -0.064
Holdout, release on or after 2020-01-01 | 348 | 0.151
Skipped, no Wednesday RWTC print | 22 | 


## Method

The EIA week ends Friday. The nominal release is the Wednesday five days later. The price is the log change from the last RWTC print before that Wednesday to the print on that Wednesday. If Wednesday has no print, the week is skipped. Thursday holiday releases are not moved.

## Result

Train correlation of surprise with the release-day RWTC log return is -0.064. Holdout correlation is 0.151. The holdout correlation is not negative. The bar was missed.